### PR TITLE
Environment setup and introduction

### PR DESCRIPTION
**Pull request by digitopolis**
Adds basic introduction to

- navigating and working with files on the command line
- setting up a new github account
- installing git and cloning project repo
- installing vs code and opening project

### CODE REVIEW

File: environment-setup/environment-setup.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+### _Extra Practice_
+Try going through the excercises in this [Codeacademy course](https://www.codecademy.com/paths/learn-how-to-build-websites/tracks/build-websites-on-your-own-computer/modules/command-line-for-building-websites/lessons/navigation/exercises/introduction)
+
+## Git and GitHub
+**Git** is a version control system – a tool that allows developers to work on different versions of the same code base at the same time. **GitHub** is a website that stores code remotely, and allows us to track changes the team has made using Git.
+
+### Getting started with GitHub
+1. Go to [github.com](https://github.com/) and click on `Sign Up`
+2. Follow the prompts to create a new account
+    - Enter your email address
```
Comment (k3ntako) [inline environment-setup/environment-setup.md:56]: Should we specify school or personal email? I'm leaning toward personal, so they can keep it long term.

File: environment-setup/environment-setup.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    - Create a username and password
+    - Verify your account (solve a puzzle?)
+3. Check your email for a 7-digit code to enter on the next screen
+4. Click 'Skip Personalization' to go to your dashboard
+5. Post in the team Slack channel with a link to your profile
+
+### Setting up Git locally
+1. [Download and install the latest version of Git](https://git-scm.com/downloads)
```
Comment (k3ntako) [inline environment-setup/environment-setup.md:64]: I've never installed Git. Have I been doing this wrong? 😅 I always thought Macs came with it.

File: environment-setup/environment-setup.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+2. Set your Git username – open Terminal and enter:
+```bash
+$ git config --global user.name "<enter your name here>"
+```
+3. Set your Git email address – in Terminal enter:
```
Comment (k3ntako) [inline environment-setup/environment-setup.md:69]: Maybe we should specify to use the one they used for Github?

**Review by k3ntako — APPROVED** [review]
This is great! It's so friendly! I left some small comments. Feel free to merge whenever you think it's ready!

**Comment by digitopolis** [comment]
Agree with all of the suggestions! Updated with those changes

**Pull request merged**